src/Filter.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+/**
+ * @property {string} variable - The variable of this field
+ */
+export default class Filter {
```

Comment:
I'm not very confident in this structure.
Here is the OpenAPI structure for filters (they did a great job): https://swagger.io/docs/specification/describing-parameters/
And the Hydra one: http://www.hydra-cg.com/spec/latest/core/#x5-2-templated-links

To support both I suggest to rename this class `Parameter` with the following structure:

```js
class Parameter {
  variable: string;
  type: string; // (an IRI)
  required: bool;
  description: string;
}
```